### DISCUSSION TITLE
Callhome support in JNC

### DISCUSSION THREAD

**Discussion by akashsi1** [tail-f-systems/JNC#144] [Q&A]
Does JNC support Callhome feature or are there any plans for adding this support ?

**Comment by martin-volf** [comment]
No, it does not support it. And no, we do not have short-term plans for adding it, though, admittedly, this is an interesting feature, we might consider adding it if time permits.

**Comment by martin-volf** [comment]
Hi @akashsi1 - we were able to add support for the callhome feature. At the time of writing this, there is #147 that implements it (feel free to comment); and the SSHJ library we are using does not support it, but a patch that adds the support has been merged. So you can use the feature already, but until new SSHJ release is available, you would have to build both SSHJ and JNC from the source.

**Comment by akashsi1** [comment]
Thanks a lot @martin-volf for such quick turn around.
